docs/themes.md

```diff
@@ -34,7 +37,7 @@ Different blocks have the possibility of customizing colors. Gutenberg provides
 
 ```php
 add_theme_support( 'gutenberg', array(
-   'colors' => array(
+	'colors' => array(
```

Comment:
Should we do the same for the colors? I mean dropping the `gutenberg` code name. Maybe something like `add_theme_support( 'colors', $colors );`

Reply:
Yeah, thinking that makes sense. Should that happen under a different issue?

Reply:
I'm fine doing it in the same PR, that way we can check for the "deprecated" theme support only once.

Reply:
Okay. Ended up doing a first run with only wide images changes/warnings first. If that's close, I'll expand it to colors.